app/assets/css/main.css

```
@@ -10,119 +10,120 @@
+
+    @font-face {
+        font-family: "Verdana";
+        src: url("../fonts/Verdana Bold.ttf");
+        font-weight: 600;
+    }
+
+    :root {
+        interpolate-size: allow-keywords;
+
+        --font-family-sans:
+            "Verdana", "Sofia Pro", "Hiragino Maru Gothic ProN", Quicksand,
```

Stylelint (17.3.0)

[error] Expected "Quicksand" to be "quicksand" (value-keyword-case)

(value-keyword-case)